educe/stac/learning/cmd/addressee.py

```diff
@@ -1,97 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-# Author: Eric Kow
-# License: CeCILL-B (French BSD3)
-
-"""
-Guess the addresee for each EDU
-"""
-
-from __future__ import print_function
-
-from educe.stac import postag, corenlp
-from educe.stac.annotation import is_edu
-from educe.stac.context import sorted_first_widest
-from educe.util import\
-    add_corpus_filters, fields_without, mk_is_interesting
-import educe.corpus
-import educe.glozz
-import educe.learning.keys
-import educe.stac
-import educe.util
-
-from ..features import\
-    players_for_doc,\
-    FeatureInput
-from ..addressee import guess_addressees
```

Comment:
I think this import plus the subsequent call were the only uses of `educe.stac.addressee.guess_addressees`. If so, you might as well remove the latter.